modules/notification/base/null.go

```diff
@@ -161,11 +162,11 @@ func (*NullNotifier) NotifyPushCommits(ctx context.Context, pusher *user_model.U
 }
 
 // NotifyCreateRef notifies branch or tag creation to notifiers
-func (*NullNotifier) NotifyCreateRef(ctx context.Context, doer *user_model.User, repo *repo_model.Repository, refType, refFullName, refID string) {
+func (*NullNotifier) NotifyCreateRef(ctx context.Context, doer *user_model.User, repo *repo_model.Repository, refFullName git.RefName, refID string) {
```

Comment:
I guess you created more conflicts with your "notification refactoring" PR.

Keeping doing so, some code would be lost sooner or later.

Could these PRs not be stale for that long? I would propose to discard that stale PR.

Reply:
The change is not too big to handle.